src/components/standalone/firewall/nat/CreateOrEditNatRuleDrawer.vue

```diff
@@ -456,6 +496,33 @@ async function saveRule() {
           :invalid-message="t(errorBag.getFirstI18nKeyFor('snat_ip'))"
           :disabled="loading.saveRule"
         />
+        <NeExpandable
+          :label="t('common.advanced_settings')"
+          :is-expanded="advancedSettingsExpanded"
+          @set-expanded="(ev: boolean) => (advancedSettingsExpanded = ev)"
+        >
+          <NeSkeleton v-if="loadingAvailableDevices" :lines="2" />
+          <NeInlineNotification
```

Comment:
Maybe it's better to put this error outside the expandable, in order to always show it if the api call fails